CHANGELOG.mdx

```diff
@@ -17,32 +17,32 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 <!-- TOWNCRIER -->
 
 ## [major.minor.patch] - 2020-09-16
-                                   
+
```

Comment:
can you merge master in your branch? I think you'll have some conflicts as this section and several lines below were removed :)